sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -164,7 +164,6 @@ SELECT interval 'interval 2 weeks 2 days 1 hour 3 minutes 2 seconds 100 millisec
 SELECT interval '2 weeks 2 days 1 hour 3 minutes 2 seconds 100 millisecond 200 microseconds';
 
 -- malformed interval literal
-select interval;
```

Comment:
Could you leave this test, please, and add one more check:
```sql
SELECT interval FROM VALUES ('abc') AS tbl(interval);
```

Reply:
I add a test to PlanParserSuite